<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SliceControlWidget</class>
 <widget class="QWidget" name="SliceControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>906</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>281</y>
     <width>891</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="totalWidget">
    <property name="sizeConstraint">
     <enum>QLayout::SetMaximumSize</enum>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>65</width>
          <height>17</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="partyNameLabel">
          <property name="text">
           <string>Party Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="ownerLabel">
          <property name="text">
           <string>Owner:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLineEdit" name="partyNameLineEdit">
          <property name="minimumSize">
           <size>
            <width>101</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>133</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ownerLineEdit">
          <property name="minimumSize">
           <size>
            <width>101</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>133</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>100</width>
        <height>17</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>100</width>
        <height>17</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <widget class="QLabel" name="orientationLabel">
          <property name="text">
           <string>Orientation:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="membersLabel">
          <property name="text">
           <string>Members:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QComboBox" name="orientationComboBox">
          <property name="minimumSize">
           <size>
            <width>81</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>131</width>
            <height>16777215</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>Radical Left</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Left</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Centre Left</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Centre</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Centre Right</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Right</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Radical Right</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="membersSpinBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>81</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>131</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>100</width>
        <height>17</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>100</width>
        <height>17</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="consensusLabel">
          <property name="text">
           <string>Consensus:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="percentageLabel">
          <property name="text">
           <string>Percentage:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QSpinBox" name="consensusSpinBox">
          <property name="minimumSize">
           <size>
            <width>61</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>131</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="percentageDoubleSpinBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>61</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>131</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhFormattedNumbersOnly</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="specialValueText">
           <string/>
          </property>
          <property name="keyboardTracking">
           <bool>true</bool>
          </property>
          <property name="showGroupSeparator" stdset="0">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>100</width>
        <height>17</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>100</width>
        <height>17</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QPushButton" name="changeColorButton">
        <property name="minimumSize">
         <size>
          <width>91</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>201</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Change Color</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removePartyButton">
        <property name="minimumSize">
         <size>
          <width>91</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>201</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Remove Party</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer_8">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>100</width>
        <height>17</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
